doc-content/enterprise-only/openshift/environment-immutable-monitoring-proc.adoc

```diff
@@ -27,12 +27,15 @@ In this command line:
 +
 . Set the following parameters as necessary:
 ** *{CENTRAL} Server Keystore Secret Name* (`{CENTRAL_CAPITAL_UNDER}_HTTPS_SECRET`): The name of the secret for {CENTRAL}, as created in <<secrets-central-create-proc>>.
+** *Smart Router Keystore Secret Name* (`KIE_SERVER_ROUTER_HTTPS_SECRET`): The name of the secret for Smart Router, as created in <<secrets-kie-create-proc>>.
 ** *Application Name* (`APPLICATION_NAME`): The name of the OpenShift application. It is used in the default URLs for {LOCAL_CENTRAL} and Smart Router. OpenShift also uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names.
 ** *Maven repository URL* (`MAVEN_REPO_URL`): A URL for a Maven repository. You must upload all the processes (KJAR files) that are to be deployed in your environment into this repository.
 ** *Maven repository username* (`MAVEN_REPO_USERNAME`): The username for the Maven repository.
 ** *Maven repository password* (`MAVEN_REPO_PASSWORD`): The username for the Maven repository.
 ** *{CENTRAL} Server Certificate Name* (`{CENTRAL_CAPITAL_UNDER}_HTTPS_NAME`): The name of the certificate in the keystore that you created in <<secrets-central-create-proc>>.
 ** *{CENTRAL} Server Keystore Password* (`{CENTRAL_CAPITAL_UNDER}_HTTPS_PASSWORD`): The password for the keystore that you created in <<secrets-central-create-proc>>.
+** *Smart Router Certificate Name* (`KIE_SERVER_ROUTER_HTTPS_NAME`): The name of the certificate in the keystore that you created in <<secrets-central-create-proc>>.
```

Comment:
Here as well, this should be "secrets-smartrouter-create-proc" to link to section 2.4